terraform/README.md

```diff
@@ -1,56 +1,91 @@
 # Terraform Provider Plugin
 
-# Installation
+## Usage
 
-1. Clone the plugin:
+Please, refer to [GETTING_STARTED guide](GETTING_STARTED.md) and [official documentation](https://goteleport.com/docs/setup/guides/terraform-provider/).
 
-```bash
-git clone git@github.com:gravitational/teleport-plugins
-```
+## Development
 
-2. Install the plugin to Teleport:
+1. Install [`protobuf`](https://grpc.io/docs/protoc-installation/).
+2. Install [`protoc-gen-terraform`](https://github.com/gravitational/protoc-gen-terraform) v1.0.0+.
 
-```bash
-cd teleport-plugins/terraform
-make install
-```
+    ```go install github.com/gravitational/protoc-gen-terraform@latest```
 
-3. Configure teleport:
+    _NOTE_: Once PR is merged, we'll replace SHA with v1.0.0
```

Comment:
`@latest` is sufficient for now
```suggestion
```

Reply:
We have not merged protoc-gen-terraform yet (